examples/daisyworld.jl

```diff
@@ -48,249 +60,269 @@ mutable struct Daisy <: AbstractAgent
     albedo::Float64 # 0-1 fraction
 end
 
+mutable struct Land <: AbstractAgent
+    id::Int
+    pos::Tuple{Int,Int}
+    temperature::Float64
+end
+
+const DaisyWorld = ABM{Union{Daisy, Land}};
+
 # ## World heating
 
 # The surface temperature of the world is heated by its sun, but daisies growing upon it
 # absorb or reflect the starlight -- altering the local temperature.
 
-function suface_temperature!(node::Int, model::ABM{Daisy})
+function update_surface_temperature!(node::Int, model::DaisyWorld)
     ids = get_node_contents(node, model)
-    absorbed_luminosity = if isempty(ids)
+    absorbed_luminosity = if length(ids) == 1 # no daisy
```

Comment:
Perhaps an explicit comment here for the user. Rather than 'no daisy', how about 'We use `fill_space!` for our `Land` agents, thus we expect to have a minimum of 1 agent in each node (i.e. when no daisy has grown here)'